docs/02_developer_install.rst

```diff
@@ -48,19 +48,19 @@ Getting started
 
 2. Activate the created vitualenv and run ``pip install -r requirements.``
 
-3. Run ``fab setup``
-    Running this command will do the following:
-
-    * Create the database.
-
-    * Run migrations.
-
-    * Load demo data if the project **DEBUG** attribute in settings is set to true .
+3. Run the following commands sequentially:
```

Comment:
Separate the setup install commands into:
1) Setup with existing database
2) Setup without existing database 
3) Recreating the database